understanding/20/headings-and-labels.html

```diff
@@ -30,6 +30,9 @@ <h2>Intent of Headings and Labels</h2>
       <p>Labels and headings do not need to be lengthy. A word, or even a single character,
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
+       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. In these cases, authors would need to ensure that the image and its use as label in a particular context can be considered conventional and widely understood. 
```

Comment:
a suggestion to cite 1.1.1, as one commenter requested:
```suggestion
       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. In these cases, authors would need to ensure that the image and its use as label in a particular context can be considered conventional and widely understood (and that the image meets 1.1.1 Non-text Content by providing a text alternative). 
```

Reply:
Per my above comment, I don’t think this addition is nuanced enough since this would imply every image itself needs alt text, but that isn’t always true